clang/test/CodeGen/sanitizer-coverage-stack-depth-debug-loc.c

```diff
@@ -0,0 +1,40 @@
+// Test that SanitizerCoverage preserves debug locations when inserting stack depth tracking
```

Comment:
This is a pure LLVM fix, and the test should not be in Clang, where we try to just test Clang's code generation and usually don't test IR lowering.

Could it be similar to e.g. `llvm/test/Instrumentation/SanitizerCoverage/missing_dbg.ll`?